temporalio/worker/_interceptor.py

```diff
@@ -314,6 +326,14 @@ async def handle_query(self, input: HandleQueryInput) -> Any:
         """Called to handle a query."""
         return await self.next.handle_query(input)
 
+    def handle_update_validator(self, input: HandleUpdateInput) -> None:
+        """Called to handle an update's validation stage."""
+        return self.next.handle_update_validator(input)
```

Comment:
```suggestion
        self.next.handle_update_validator(input)
```

(pedantic, doesn't really matter)